tests/build-and-install-rpms.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Why did this need to change? `#!/bin/bash` works fine?

Reply:
I run on macOS.
On macOS system `/bin/bash` (version 3.2) is often replaced with a newer `bash` in `/usr/local/bin/bash`(which has version 5.0.18).

Reply:
I'll revert this nevertheless, since this error:
https://travis-ci.com/github/cobbler/cobbler/jobs/449646165#L9817
is not happening on my machine.

Reply:
Makes no difference... 😢